diff --git a/CMakeLists.txt b/CMakeLists.txt
index 427015be..e945a722 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1157,6 +1157,7 @@ endif()
 #
 
 add_subdirectory(common)
+add_subdirectory(examples/main)
 
 if (LLAMA_BUILD_TESTS AND NOT CMAKE_JS_VERSION)
     include(CTest)
diff --git a/examples/main/main.cpp b/examples/main/main.cpp
index 0ed4d79f..74b77cc9 100644
--- a/examples/main/main.cpp
+++ b/examples/main/main.cpp
@@ -911,6 +911,9 @@ int main(int argc, char ** argv) {
         llama_save_session_file(ctx, path_session.c_str(), session_tokens.data(), session_tokens.size());
     }
 
+    printf("\n");
+    fflush(stdout);
+
     llama_print_timings(ctx);
     write_logfile(ctx, params, model, input_tokens, output_ss.str(), output_tokens);
 
